modules/aws_vpn/tf_module/main.tf

```diff
@@ -0,0 +1,74 @@
+data "aws_vpc" "vpc" {
+  id = var.vpc_id
+}
+
+data "aws_kms_key" "env_key" {
+  key_id = var.kms_account_key_arn
+}
+
+resource "aws_security_group" "vpn" {
+  name_prefix = "opta-${var.env_name}-vpn"
+  description = "VPN security group."
+  vpc_id      = var.vpc_id
+
+  tags = {
+    "Name" = "opta-${var.env_name}-vpn"
+  }
+
+  egress {
+    description = "alloutbound"
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    # To be fixed - for now user can create an SG manually to override.
+    #tfsec:ignore:aws-vpc-no-public-egress-sgr
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
We could change this to default to the VPC CIDR blocks and allow the user to override to `0.0.0.0/0` if they want to use the VPN for internet access. That would get rid of the tfsec warning.

Reply:
we're currently hard coding split tunneling so I'm just setting this to the vpc cidr